robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -20,3 +22,62 @@ def _is_current_page(self):
+        locator_default = eda_lex_locators["eda_settings_cc"]["default_cc_checkbox"]
+        locator_edit_mode = eda_lex_locators["eda_settings_cc"]["enable_cc_checkbox"]
+
+        self.selenium.wait_until_page_contains_element(locator_default)
+        actual_value = self.selenium.get_webelement(locator_default).get_attribute("alt")
+        if not actual_value == "False":
+            self.eda.click_edit_on_eda_settings_page()
+            self.selenium.wait_until_page_contains_element(
+                locator_edit_mode, error=f"Enable course connections checkbox with locator '{locator_edit_mode}' is not available on the page")
+            self.selenium.click_element(locator_edit_mode)
+            self.eda.click_action_button_on_eda_settings_page("Save")
+
+    def verify_dropdown_values(self, field, *args):
+        """ Verify the field has the drop down values given by *args
+            we have to pass all the values available in the UI since the count is also verified
+        """
+        locator = eda_lex_locators["eda_settings_cc"]["dropdown_values"]
+        locator_count = eda_lex_locators["eda_settings_cc"]["dropdown_values_count"].format(field)
+        actual_count = int(self.selenium.get_element_count(locator_count))
+        expected_count = args.__len__()
+        if expected_count == actual_count:
+            for value in args:
+                locator = locator.format(field, value)
+                self.selenium.page_should_contain_element(locator, message=f"'{field}' does not contain the value '{value}'")
+        else:
+            raise Exception(f"Mismatch in count of drop down values. Actual = '{actual_count}'. Expected = '{expected_count}'")
+
+    def verify_enable_course_connections_warning(self, is_displayed):
+        """ Verify the warning message is displayed
+            this message gets displayed when the 'Enable Course Connections' field is unchecked
+        """
+        locator_enabled = eda_lex_locators["eda_settings_cc"]["enable_cc_warning_enabled"]
+        locator_disabled = eda_lex_locators["eda_settings_cc"]["enable_cc_warning_disabled"]
+        if str(is_displayed).lower() == "true":
+            self.selenium.wait_until_page_contains_element(locator_enabled,
+                                                           error="Enable course connections warning is not displayed")
+        else:
+            self.selenium.wait_until_page_contains_element(
+                locator_disabled, error="Enable course connections warning is displayed")
+
+    def verify_enable_course_connections(self, expectedCheckboxValue):
+        """ This method will check the default value of 'Enable Course Connections' checkbox
+        """
+        locator_default = eda_lex_locators["eda_settings_cc"]["default_cc_checkbox"]
+        actual_value = self.selenium.get_webelement(locator_default).get_attribute("alt")
+        if not str(expectedCheckboxValue).lower() == "true" :
```

Comment:
shouldn't you be comparing the actual value and expected value here?

Reply:
@skristem It is and it should be else it is always set to true. Updated it and thanks for pointing it out